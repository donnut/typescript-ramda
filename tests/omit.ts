import R_omit = require('../ramda/dist/src/omit');

declare const object: object;
declare const string: string;

// @dts-jest:pass:snap
R_omit([string])(object);
// @dts-jest:pass:snap
R_omit([string], object);

// @dts-jest:pass:snap
R_omit([string])(object);
// @dts-jest:pass:snap
R_omit([string], object);

// @dts-jest:pass:snap
R_omit(['a', 'b'], { a: 1, b: 2, c: 3 });
// @dts-jest:pass:snap
R_omit(['a', 'b'])({ a: 1, b: 2, c: 3 });
