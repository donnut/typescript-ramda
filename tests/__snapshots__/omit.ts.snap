// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`R_omit(['a', 'b'])({ a: 1, b: 2, c: 3 }) 1`] = `"Partial<{ a: number; b: number; c: number; }>"`;

exports[`R_omit(['a', 'b'], { a: 1, b: 2, c: 3 }) 1`] = `"Pick<{ a: number; b: number; c: number; }, \\"c\\">"`;

exports[`R_omit([string])(object) 1`] = `"object"`;

exports[`R_omit([string])(object) 2`] = `"object"`;

exports[`R_omit([string], object) 1`] = `"object"`;

exports[`R_omit([string], object) 2`] = `"object"`;
